frontend/guardian.rb

```diff
@@ -29,31 +28,24 @@ def restart(command)
   return tag_reporter
 end
 
-def close_door(gpio_command)
-  `#{gpio_command} -g write 9 1`
-end
-
-def open_door(gpio_command)
-  `#{gpio_command} -g write 9 0`
+def close_door(gpio_command, door)
```

Comment:
Suggestion: since the rollup doesn't actually close when the pin goes low, (putting the pin low on the rollup will be the equivalent of taking your finger off the open button on the existing control panel.  The roll up will continue opening and will stop fully open.) I suggest you don't reuse the previous methods and instead split off different ones, or rename the methods to accurately convey their functionality.